rviz_ogre_vendor/CMakeLists.txt

```diff
@@ -40,8 +40,7 @@ macro(build_ogre)
     TIMEOUT 600
     CMAKE_ARGS
       -DOGRE_STATIC:BOOL=ON
-      -DOGRE_BUILD_DEPENDENCIES:BOOL=OFF
-      -DOGRE_DEPENDENCIES_DIR=${CMAKE_CURRENT_SOURCE_DIR}/ogredeps
```

Comment:
What this necessary, or could we set this to something in the binary directory? I now get new files in my `src` folder after doing a build, e.g.:

```
Untracked files:
  (use "git add <file>..." to include in what will be committed)

	rviz_ogre_vendor/ogredeps/
```

We try to strive for "source folders are read-only while building". Is it possible you can look into how to address this?

Reply:
I removed the line for the same reasons (The build was placing dependencies in the source folder).
However, since dependencies are now built and installed together with ogre it shouldn't be necessary to download any dependencies at all...

I'll look into it.